node/cmd/guardiand/admintemplate.go

```diff
@@ -1252,3 +1273,16 @@ func isValidUint256(s string) (bool, error) {
 
 	return true, nil
 }
+
+// randSeqNonce returns a new random sequence and nonce. It exits if the random numbers can't be generated.
+func randSeqNonce() (uint64, uint32) {
+	seq, err := random.Uint64()
+	if err != nil {
+		log.Fatal("random number: ", err)
```

Comment:
Not sure it matters, but can you make the log text unique between these two cases.

Reply:
The log text will appear as `"random number: failed to generate random <type>: <actual error>"` due to the formatting in `Error()`, so the complete log will distinguish which function failed.

Is that OK?